packages/compass-schema/src/components/export-schema-modal.tsx

```diff
@@ -177,14 +190,16 @@ const ExportSchemaModal: React.FunctionComponent<{
         <Button onClick={onClose} variant="default">
           Cancel
         </Button>
-        <Button
-          onClick={() => {
-            /* TODO(COMPASS-8704): download and track with trackSchemaExported */
-          }}
-          variant="primary"
-        >
-          Export
-        </Button>
+        <Link download={filename} href={downloadUrl} onClick={onExportedSchema}>
+          <Button
+            variant="primary"
+            isLoading={!downloadUrl}
+            loadingIndicator={<SpinLoader />}
+            disabled={!exportedSchema}
+          >
+            Export
+          </Button>
+        </Link>
```

Comment:
Leafygreen buttons support being rendered as links out of the box

```suggestion
          <Button
            variant="primary"
            isLoading={!downloadUrl}
            loadingIndicator={<SpinLoader />}
            disabled={!exportedSchema}
             download={filename}
             href={downloadUrl}
             onClick={onExportedSchema}
          >
            Export
          </Button>
```

Reply:
Ah thanks! ❤️